ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -84,15 +84,28 @@ def training_behaviors(self) -> Dict[BehaviorName, BehaviorSpec]:
     def close(self):
         pass
 
-    def advance(self):
+    def advance(self) -> int:
```

Comment:
Since this isn't a user-facing method, I think we can delete it instead of deprecating it. But I'm OK either way.

Reply:
Sounds good. I was partly being lazy since a few unit tests use it, but I should just clean it up.

Reply:
Removed.